examples/game/breakout.rs

```diff
@@ -1,16 +1,54 @@
+//! A simplified implementation of the classic game "Breakout"
+
 use bevy::{
     core::FixedTimestep,
+    math::{const_vec2, const_vec3},
     prelude::*,
     sprite::collide_aabb::{collide, Collision},
 };
 
-/// An implementation of the classic game "Breakout"
+// Sets the amount of time that should elapse between each physics step
 const TIME_STEP: f32 = 1.0 / 60.0;
+
+// These constants are in `Transform` units:
+// with the default 2D camera they correspond 1:1 with screen pixels
+// These macros are needed as functions that operate on floats cannot be constant in Rust :(
```

Comment:
```suggestion
// The `const_vec3!` macros are needed as functions that operate on floats cannot be constant in Rust.
```
A little more specific and remove the frowny face as it feels like you're disparaging rust, rather than a cheeky comment.